src/main/java/com/spikes2212/command/drivetrains/swerve/commands/TurnModules.java

```diff
@@ -0,0 +1,80 @@
+package com.spikes2212.command.drivetrains.swerve.commands;
+
+import com.spikes2212.command.drivetrains.swerve.SwerveDrivetrain;
+import com.spikes2212.command.drivetrains.swerve.SwerveModule;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/**
+ * A command that moves each {@link SwerveModule} in a given {@link SwerveDrivetrain} to a certain angle using pid to
+ * position for each one of the {@link SwerveModule}s.
+ *
+ * @author Gil Ein-Gar
+ * @see SwerveDrivetrain
+ */
+public class TurnModules extends Command {
```

Comment:
I can't believe I haven't realized that this does not have an `isFinished` condition